openquakeplatform_ipt/views.py

```diff
@@ -406,7 +406,7 @@ def view(request, **kwargs):
         request,
         "ipt/ipt.html",
         dict(
-            g_gmpe=gmpe,
+            g_gmpe=[str(x) for x in gmpe],
```

Comment:
No, this does not work unless you are sure `x` cannot contain non-ASCII characters.

Reply:
BTW, if GMPE is the name of a GMPE it is guaranteed to be pure ASCII, but I would add a comment anyway.